data/hai/unfettered missions.txt

```diff
@@ -0,0 +1,3285 @@
+	to complete
+		has "Unfettered: Alphas 3: Bring The Spoils: offered"
+	on complete
+		log `Stole an Alpha auxiliary with its cargo of jump drives, and also found Korath prisoners on board. The Alphas had a trap set for me, but I got the upper hand. I wonder if the Unfettered were in on this, considering it was through them I learned about the fuel leak. Hopefully not...`
+		outfit "Jump Drive" -10
+		payment 10000000
+		"salary: Unfettered" = 1000
+		conversation
+			`As you land with the Auxiliary you notice tens, then hundreds Unfettered start coming your way, cheering. They are probably expecting jump drives, and more keep coming from every direction. Maybe they have not seen this Auxiliary yet, but only heard tales of it, or maybe this is what always happens when it lands, loaded with jump drives. Whatever the case, they seem surprised when they see you exiting the ship. They murmur with each others and then they just start saluting, by rising one fist up. First a few do it, then more, until finally all of them are there, saluting in front of you in respect. You notice they are doing different signs with their hand.`
+			choice
+				`	(Raise my fist like warriors do.)`
+				`	(Bow.)`
+					goto bow
+				`	(Do nothing.)`
+					goto stadium
+			`	They approve of your risen fist, and seem happy you understand their culture.`
+				goto stadium
+			label bow
+			`	It does not seem to be the right reaction at all. They shout at you, all form of respect is gone. One very shocked Unfettered soldier even tries to shoot you for this insult! It looks like you have shown yourself to be weak to them. After some time a Warlord arrives and calms them down.`
+			label stadium
+			`	The crowd lead you to the stadium.`
+			`	The council seems surprised to see you return, especially the Unfettered warlord that annoyed you earlier, but they address you with a deeper respect than before. "You have proven yourself. Now we have almost enough jump drives for the next phase..." You look at them intensively, and after a few moments they realize, "Oh right, your payment! Here it is : <payment>!"`
+			choice
+				`	"I have found Korath prisoners on the Alpha ship."`
+			`They talk about it and then one of the Unfettered elders speak up, "Perfect, hand them over to us, we will see to it they have all the comfort they need, hopefully in exchange for some technology." They discuss among themselves again, then the warlord from earlier speaks, seemingly with a heavy burden. "Now that you officially work for us, you will have a salary of 1000. We will not make you wear our colors yet because it could cause problems with the Hai. We do not want you to get shot on sight."`
+			branch logs
+				has "Unfettered: alpha logs"
+			decline
```

Comment:
there is some weird stuff going on with these sometimes, yeah you're right this will literally write 'decline' on the screen
I had to do the same fake text indentation at other places, it does work